lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1559,6 +1561,7 @@ def _resource_sku_to_capability(  # noqa: C901
             [
                 schema.FeatureSettings.create(features.StartStop.name()),
                 schema.FeatureSettings.create(features.SerialConsole.name()),
+                schema.FeatureSettings.create(features.Hibernation.name()),
```

Comment:
Does it mean all vm size support it?

Reply:
from the spec, it has some recommended sizes, I will figure out whether any API expose this.

Reply:
It's ok, if there is no API about it. We can just list recommended vm families.